### ISSUE TITLE
New paper which claimed it solves all cubes they generated 

### ISSUE THREAD

**Issue by dusmart** [jasonrute/puzzle_cube#1]
Solving the Rubik's Cube Without Human Knowledge
Stephen McAleer, Forest Agostinelli, Alexander Shmakov, Pierre Baldi
(Submitted on 18 May 2018)

**Comment by jasonrute** [comment]
Yes, I've have been meaning to do something about this.  The biggest difference between that paper and mine is that in their tests, they let the solver run for 1 hour.  In my tests, I only let it run a few seconds.  I was under (the probably false) assumption that if it couldn't find a solution in a few seconds, it couldn't find one at all.  I need to try out longer solve times.

There are, of course, other differences in the way the cube is represented, the way it is trained, and the type of neural network.  However, if I were to guess, the amount of time given to solve a cube is the largest difference.

**Comment by dusmart** [comment]
Thanks for replying! I'm really happy.

Yeah. As you say, we just didn't think about the long time running for finding a solution.
Actually, I'm a freshman in this area. I haven't even try your solution. I asked them for the public code two days ago. But it seems that they don't want to lease the code. 

Please update your Github if good results come. I'm trying hard to understand your idea.

Thanks for your great project!
